onionshare/web/share_mode.py

```diff
@@ -16,9 +16,8 @@ class ShareModeWeb(SendBaseModeWeb):
     """
     def init(self):
         self.common.log('ShareModeWeb', 'init')
-        # If "Stop sharing after files have been sent" is unchecked and "Allow downloading of individual files" is checked
-        self.download_individual_files = not self.common.settings.get('close_after_first_download') \
-            and self.common.settings.get('share_allow_downloading_individual_files')
+        # Allow downloading individual files if "Stop sharing after files have been sent" is unchecked
+        self.download_individual_files = not self.common.settings.get('close_after_first_download')
```

Comment:
@micahflee we might not need this overridden `init()` class at all now? I can't see where this attribute `self.download_individual_files` is evaluated now.

Reply:
No I think we still need it. It gets used later in the `directory_listing_template` and `render_logic` methods, basically to tell the template if the filename should be a link, and to prevent people from downloading the file if they just type in the URL, even though it's not a link.

Reply:
Yep my bad, I was viewing an individual commit instead of the full diff :sigh: